src/app/views/App.tsx

```diff
@@ -254,6 +258,9 @@ class App extends Component<IAppProps, IAppState> {
     const properties = { ...sidebarProperties };
     properties.showSidebar = !properties.showSidebar;
     this.props.actions!.toggleSidebar(properties);
+    this.setState(state => ({
+      onToggleSidebar: !this.state.onToggleSidebar
+    }));
```

Comment:
The global state has the state of the sidebar toggle. Creating a local state variable to capture this will be redundant and may have inconsistent results. The redux state is a better source of truth